docs/architecture/README.md

```diff
@@ -160,34 +158,33 @@ The Page Rendering is therefore delegated to the corresponding App Cell instance
 - --> [App Instantiation](#app-instantiation)
 - --> [App Rendering](#app-rendering)
 
-
 **b) only_page**
 
-This render_mode applies, if ONE OF the following conditions are met:
+This `render_mode` applies if ONE OF the following conditions is met:
 
-- Explicit only_page request
+- Explicit `only_page` request
   - the page is requested with the query param: `only_page=true`
   - (these requests are used for dynamic transitions)
 - Page Cell is NOT related to an App Cell
   - --> `Pages::MyFirstPage` --> not related to an App Cell
 
-If one of the conditions are met, the page.haml template is used in order add some markup
+If one of the conditions is met, the page.haml template is used to add `div`-wrapper:
 
 ```HAML
 %div{class: "matestack_page_content"}
   - @cells.each do |key, cell|
     = cell.call(:show)
 ```
 
-and to iterate through all Component Cell instances stored in `@cells`, call show on each of them and render their response to HTML.
+The Page then iterates through all Component Cell instances stored in `@cells`, calls show on each of them and renders their response to HTML.
 
 - --> [Component Rendering](#component-rendering)
 
 **c) only_component**
 
-This render_mode applies, if the following condition is met:
+This `render_mode` applies if the following condition is met:
 
-- Explicit only_component request
+- Implicit `only_component` request
```

Comment:
if you call a route with a specific query param, it is 'explicit' in my opinion

Reply:
I did non find `only_component` elsewhere in the core, which made it kind-of-implicit for me. Maybe we can use another half sentence to make things clearer?

Reply:
I've slightly changed the wording to make it more obvious in my latest commit on this branch!